### PR TITLE
docextract: update report-numbers.kb

### PR DESCRIPTION
**Pull request by fschwenn**
* Adds some CMS report numbers.

Signed-off-by: fschwenn <florian.schwennsen@desy.de>

### CODE REVIEW

File: refextract/references/kbs/report-numbers.kb [hunk 1]
```diff
@@ -268,13 +268,20 @@ CMS PAS EWK            ---CMS-PAS-EWK
 CMS PAS BTV            ---CMS-PAS-BTV
 CMS PAS FWD            ---CMS-PAS-FWD
 CMS PAS TRK            ---CMS-PAS-TRK
-CMS PAS SMP            ---CMS-PAS-SMP
```
Comment (michamos) [inline refextract/references/kbs/report-numbers.kb]: was this deleted on purpose?

Comment (tsgit) [reply refextract/references/kbs/report-numbers.kb]: yes, it's a dup from line 265

**Comment by tsgit** [comment]
LGTM

File: refextract/references/kbs/report-numbers.kb [hunk 2]
```diff
@@ -268,13 +268,20 @@ CMS PAS EWK            ---CMS-PAS-EWK
...
 CMS PAS PFT            ---CMS-PAS-PFT
 CMS PAS MUO            ---CMS-PAS-MUO
 CMS PAS JME            ---CMS-PAS-JME
 CMS PAS EGM            ---CMS-PAS-EGM
 CMS PAS DIF            ---CMS-PAS-DIF
 CMS PAS B2G            ---CMS-PAS-B2G
+CMS PAS LUM            ---CMS-PAS-LUM
+CMS PAS FTS            ---CMS-PAS-FTA
```
Comment (tsgit) [inline refextract/references/kbs/report-numbers.kb]: is this correct, changing FTS to FTA ?

Comment (tsgit) [reply refextract/references/kbs/report-numbers.kb]: I note that neither `CMS-PAS-FTA` nor `CMS-PAS-FTS` do appear to be used. See http://cdsweb.cern.ch/collection/CMS%20Physics%20Analysis%20Summaries?ln=en&as=1

search for those report number prefixes turns up empty

**Comment by tsgit** [comment]
closing this PR, superseded by 
https://github.com/inspirehep/refextract/pull/66

**Pull request closed without merging**